File: src/lib/models/DTO/Domain/HostDto.cs
using DTO.Domain.Ext.Zoom;
using System.Collections.Generic;

namespace DTO.Domain
{
    public class HostDto : BaseDomainDto
    {
        public string Company{ get; set; }
        public string CareerSummary{ get; set; }
        public long ExtUserId{ get; set; }
        public bool? IsConsultant{ get; set; }
        public ICollection<FocusHostMappingDto> Specialities{ get; set; }

        public ExtZoomUserDto ExtUser{ get; set; }
    }
}

File: src/ui/asq-ui/src/app/_models/domain/host-dto.ts
import { BaseDomainDto } from './base-domain-dto';
import { FocusHostMappingDto } from './focus-host-mapping-dto';

export class HostDto extends BaseDomainDto{
    public company: string = null;
    public careerSummary: string = null;
    public isConsultant: boolean = false;
    public specialities: FocusHostMappingDto[] = null;

    constructor(){ super(); }
}
